src/Universe.cpp

```diff
@@ -1194,13 +1211,15 @@ Cell* Lattice::findCell(LocalCoords* coords) {
   /* Create a new LocalCoords object for the next level Universe */
   LocalCoords* next_coords;
 
-  if (coords->getNext() == NULL)
+  if (coords->getNext() == NULL) {
     next_coords = new LocalCoords(next_x, next_y, next_z);
+  }
```

Comment:
Nope, not needed now though they were for an early implementation. I'll remove them now.